src/backend/revoker/indexer/queryAttestations.ts

```diff
@@ -0,0 +1,126 @@
+import {
+  IAttestation,
+  type DidUri,
+  type HexString,
+  type ICType,
+} from '@kiltprotocol/sdk-js';
+
+import { wholeAttestation, wholeBlock } from './fragments';
+import { matchesGenerator, QUERY_SIZE } from './queryFromIndexer';
+
+// When modifying queries, first try them out on https://indexer.kilt.io/ or https://dev-indexer.kilt.io/
+
+function buildAttestationQueries(
+  issuedBy: DidUri,
+  fromDate: Date,
+  untilDate: Date,
+) {
+  return (offset: number) => `
+  query {
+  attestations(
+    orderBy: ID_ASC
+    first: ${QUERY_SIZE}
+    offset: ${offset}
+    filter: {
+      issuerId: {
+        equalTo: "${issuedBy}"
+      }
+      creationBlock: { timeStamp: { greaterThan: "${fromDate.toISOString()}", lessThan: "${untilDate.toISOString()}"  } }
```

Comment:
```suggestion
      creationBlock: {
        timeStamp: { 
          greaterThan: "${fromDate.toISOString()}", 
          lessThan: "${untilDate.toISOString()}"  
        } 
      }
```
Would it be consistent with the formatting above and also more readable this way?

Reply:
Sure.

Reply:
done